spec/system/admin/reports_spec.rb

```diff
@@ -402,8 +402,6 @@
     before do
       product1.set_property 'Organic', 'NASAA 12345'
       product2.set_property 'Organic', 'NASAA 12345'
-      product1.taxons = [taxon]
-      product2.taxons = [taxon]
```

Comment:
Hum, this spec works on the factory data for primary taxon. 
Shrug, not worth  updating.